packages/workbox-core/_private/cacheNames.mjs

```diff
@@ -9,11 +9,11 @@
 import '../_version.mjs';
 
 const _cacheNameDetails = {
-  prefix: 'workbox',
-  suffix: self.registration.scope,
   googleAnalytics: 'googleAnalytics',
-  precache: 'precache',
+  precache: 'precache-v4',
```

Comment:
Right. Another option is `'precache-v2'`, implying that it's the second precache version, rather than using the version number associated with Workbox.

Would you prefer that?

Reply:
If you're following the guidance at https://developers.google.com/web/tools/workbox/guides/configure-workbox#configure_cache_names and changing `prefix`, then you'll automatically get the new `'precache-v2'` as part of your cache name.

If you're bypassing that cache name mechanism entirely (which... I'm assuming not too many people are, since you have to use `PrecacheController` directly vs. using `workbox.precaching`) then your cache name will not end up changing. As for how bad a problem that is... I *think* the worst case scenario is ending up with extra entries in the named cache that would never be cleaned up, because the new `workbox-precaching` code isn't tracking them.

Reply:
Cool. Probably worth mentioning in the upgrade notes that anyone in that latter case should probably choose a new name for v4.